CSS/style.css

```diff
@@ -0,0 +1,913 @@
+  }
+
+  .par-logo {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+    gap: 30px;
+    font-size: 28px;
+    color: gray;
+  }
+
+  .par-logo li:first-child,
+  .par-logo li:nth-child(3) {
+    font-weight: 800;
+    color: gray;
+  }
+
+  .par-logo li:nth-child(2) {
+    font-weight: 800;
+    color: #9b9999;
+  }
+
+  .par-logo li:nth-child(4) {
+    font-weight: 300;
+    color: gray;
+  }
+
+  .par-logo li:last-child {
+    color: #6a6969;
+  }
+
+  .footer-end {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 15px;
+    margin: 20px auto;
+    width: 70%;
+  }
+
+  .foot-logo {
+    width: 180px;
+  }
+
+  /* Start about page  */
+  .sect-two {
+    padding-bottom: 50px;
+    border-bottom: 1px solid #b8b6b6;
+  }
+
+  .sect-two h2 {
+    font-size: 28px;
+    text-align: center;
+    padding: 10px 10px 22px 10px;
+    width: 100%;
+    margin: 40px auto;
+    position: relative;
+  }
+
+  .sect-two h2::before {
+    content: '';
+    width: 50px;
+    height: 2px;
+    position: absolute;
+    background-color: var(--main-color);
+    left: 40%;
+    bottom: 0;
+  }
+
+  .sect-logo {
+    text-align: center;
+    margin: 40px auto 0 auto;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .sect-thre h2 {
+    font-size: 28px;
+    text-align: center;
+    padding: 10px 10px 22px 10px;
+    width: 100%;
+    margin: 20px auto 40px auto;
+    position: relative;
+  }
+
+  .sect-thre h2::before {
+    content: '';
+    width: 50px;
+    height: 2px;
+    position: absolute;
+    background-color: var(--main-color);
+    left: 40%;
+    bottom: 0;
+  }
+
+  .sect-thre-cont {
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+    margin-top: 30px;
+  }
+
+  .cont-img {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 10px;
+    color: #fff;
+    width: 100%;
+    height: 300px;
+    background-image: url(/images/conf.PNG);
+    position: relative;
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .cont-img::before {
+    content: '';
+    width: 100%;
+    height: 100%;
+    background-color: rgb(236 31 21 / 84%);
+    opacity: 0.7;
+    position: absolute;
+  }
+
+  .logo-footer > img {
+    width: 100px;
+  }
+
+  /* End about page  */
+
+  .logo {
+    display: none;
+  }
+
+  .nav-top {
+    display: none;
+  }
+
+  .fa-3x {
+    display: none;
+  }
+}
+
+@media (min-width: 769px) {
+  .hamburger {
+    display: none;
+  }
+
+  .hidden {
+    display: none;
+  }
+
+  .container {
+    width: 90%;
+    padding: 10px;
+    margin: 0 auto;
+    max-width: 1440px;
+  }
+
+  .nav-bar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    position: fixed;
+    background-color: #fff;
+    width: 90%;
+    height: 65px;
+    z-index: 5;
+    top: 40px;
+    padding-left: 10%;
```

Comment:
- [ ] Hey @codehass you really did a great job on this project 👏. However, I have a few suggestions to make it nicer.
- [ ]  First I would kindly recommend removing the `padding-left: 10%;` on the `nav-bar` so that the `logo` is going to be aligned to the right with the content of the hero section like in the design. And the `padding-left` reduce it to `24px` instead. 😄
- [ ] Second please for a better user experience make sure that if someone clicks on the logo in the nav bar, it leads him or her to the home page.  👌